composer.json

```diff
@@ -41,11 +41,6 @@
     "config": {
         "sort-packages": true
     },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "1.x-dev"
```

Comment:
This repository have not `1.x` branch. IMO we should not merge `remove branch alias` PR here.

Reply:
This package is not even used https://packagist.org/packages/sonata-project/classification-media-bundle/stats

The rule I followed is to remvove the alias on every unstable branch. This "BC-break" seems acceptable to me